### PR TITLE
LPS-142295 Provide vanilla-counter custom element as default sample remote app

### PR DESCRIPTION
**Pull request by izaera**
This PR includes:

- https://issues.liferay.com/browse/LPS-142295
- https://issues.liferay.com/browse/LPS-142129

Note that the vanilla-counter custom element pointed by the sample data is not yet available (we need IT to publish it).

### CODE REVIEW

**Comment by izaera** [comment]
Since LPS-142129 is a fix for a bug, and LPS-142295 was requested directly by Brian, I'm going to forward this without further aproval.

**Comment by izaera** [comment]
ci:forward

**Comment by nhpatt** [comment]
Spice must flow

File: modules/apps/remote-app/remote-app-web/src/main/java/com/liferay/remote/app/web/internal/display/context/EditRemoteAppEntryDisplayContext.java [hunk 1]
```diff
@@ -100,15 +106,24 @@ public String getCmd() {
 			_portletRequest, "customElementURLs", customElementURLs);
 
 		if (customElementURLs.length == 0) {
-			customElementURLs = new String[1];
+			customElementURLs = new String[] {
+				"https://liferay.github.io/liferay-frontend-projects" +
```
Comment (nhpatt) [inline modules/apps/remote-app/remote-app-web/src/main/java/com/liferay/remote/app/web/internal/display/context/EditRemoteAppEntryDisplayContext.java:110]: you are going to send it before it's enabled :O

**Review by nhpatt — APPROVED** [review]

File: modules/apps/remote-app/remote-app-web/src/main/java/com/liferay/remote/app/web/internal/display/context/EditRemoteAppEntryDisplayContext.java (hunk 1 above)
Comment (izaera) [reply modules/apps/remote-app/remote-app-web/src/main/java/com/liferay/remote/app/web/internal/display/context/EditRemoteAppEntryDisplayContext.java:110]: Yes. I said it in the PR description.

**Comment by izaera** [comment]
I will resend two PRs, so that LPS-142295 is not immediately forwarded.

File: modules/apps/remote-app/remote-app-web/src/main/java/com/liferay/remote/app/web/internal/display/context/EditRemoteAppEntryDisplayContext.java (hunk 1 above)
Comment (izaera) [reply modules/apps/remote-app/remote-app-web/src/main/java/com/liferay/remote/app/web/internal/display/context/EditRemoteAppEntryDisplayContext.java:110]: See https://github.com/liferay-frontend/liferay-portal/pull/1654#issuecomment-966316823

**Comment by izaera** [comment]
https://github.com/liferay-frontend/liferay-portal/pull/1655
https://github.com/liferay-frontend/liferay-portal/pull/1656

**Pull request closed without merging**